Framework/PythonInterface/mantid/geometry/src/Exports/DetectorInfoPythonIterator.cpp

```diff
@@ -20,22 +19,10 @@ void export_DetectorInfoPythonIterator() {
   // Export to Python
   class_<DetectorInfoPythonIterator>("DetectorInfoPythonIterator", no_init)
       .def("__iter__", objects::identity_function())
-      .def(
-#if PY_VERSION_HEX >= 0x03000000
-          "__next__"
+#ifdef IS_PY3K
```

Comment:
Would it make sense to test against Python 2.x here, leaving the `#else` clause to deal with all later Python versions? A bit of discussion on the topic can be found [here](http://astrofrog.github.io/blog/2016/01/12/stop-writing-python-4-incompatible-code/).